### PR TITLE
NSV Materials Fix

### PR DESCRIPTION
**Pull request by Ikalpo**
## About The Pull Request

makes the NSV material amounts look nicer in code
fixes the durasteel sheet material amount (it was missing a bit of titanium)
fixes the amount NSV materials cost in ORMs and lathes
adds material amounts to `tile/carpet/ship` (they are made of durasteel)

## Why It's Good For The Game

fix man good
and also I want to be able to recycle those floor tiles when I accidently make them

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

image

</details>

## Changelog
:cl:
fix: You can now recycle nanoweave carpet tiles
fix: Fixed the amount that NSV materials cost in ORMs and lathes
/:cl:

### CODE REVIEW

**Comment by Karmic-Skink** [comment]
image

The implication that the green there looks nicer in code is 100% subjective and also 1000% the wrong choice.

**Comment by Bokkiewokkie** [comment]
> image
> 
> The implication that the green there looks nicer in code is 100% subjective and also 1000% the wrong choice.

Yeah @Ikalpo you should just make it one decimal or fraction

**Comment by Ikalpo** [comment]
> > image
> > The implication that the green there looks nicer in code is 100% subjective and also 1000% the wrong choice.
> 
> Yeah @Ikalpo you should just make it one decimal or fraction

is `list(/datum/material/iron = MINERAL_MATERIAL_AMOUNT*0.2/4, /datum/material/silver = MINERAL_MATERIAL_AMOUNT*0.15/4, /datum/material/titanium = MINERAL_MATERIAL_AMOUNT*0.65/4)` better?

**Comment by Karmic-Skink** [comment]
> > > image
> > > The implication that the green there looks nicer in code is 100% subjective and also 1000% the wrong choice.
> > 
> > 
> > Yeah @Ikalpo you should just make it one decimal or fraction
> 
> is `list(/datum/material/iron = MINERAL_MATERIAL_AMOUNT*0.2/4, /datum/material/silver = MINERAL_MATERIAL_AMOUNT*0.15/4, /datum/material/titanium = MINERAL_MATERIAL_AMOUNT*0.65/4)` better?

No

**Comment by Karmic-Skink** [comment]
Okay so firstly, you are making this far more complicated than it needs to be. It really does not need to contain any operations to determine what the exact number needs to be and just makes it harder to read.

image
Consider this image.
When we are taught mathematics at school, we are first taught fractions as these are the easiest numbers for our human brains to make sense of.
As we progress, we move forward into decimals and scientific nomenclature.
Which most adults have somewhat an understanding of.

By adding basically a horrible inverse of a fraction using decimals and including operators, you are effectively making it extremely hard to read for no real reason.

tl;dr there is absolutely nothing wrong with using fractions or straight decimals, please stop trying to reinvent the wheel.

**Review by Bokkiewokkie — CHANGES_REQUESTED** [review]
Agree with Karmic on the readability of multiplications here (btw karmic just request changes for it next time)

**Review by Bokkiewokkie — APPROVED** [review]
